charts/postiz/README.md

```
@@ -0,0 +1,671 @@
+      REPLICA_COUNT: "3"
+```
+
+## Accessing the Application
+
+### Port Forwarding
+
+**Postiz Application:**
+```bash
+kubectl port-forward svc/postiz-app 3000:80
+```
+Visit http://localhost:3000
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)